core/scripts/keystone/src/05_deploy_initialize_capabilities_registry.go

```diff
@@ -259,6 +286,8 @@ func (c *deployAndInitializeCapabilitiesRegistryCommand) Run(args []string) {
 		panic(err)
 	}
 
+	//TODO(mstreet3): Add custom compute action capability
```

Comment:
Correct :)

The LabelledName should be "custom_compute" and the version should be "1.0.0". It's an action, uint8(1) (I think)